.github/workflows/code-quality.yaml

```diff
@@ -9,25 +9,26 @@ jobs:
   code-static-analysis:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4.1.1
```

Comment:
I'd keep it @v4 and let it auto update

Reply:
Right, wasn't aware - https://github.com/orgs/community/discussions/48058 . Thanks, I'll update.